src/components/sales-report/index.js

```diff
@@ -1,12 +1,24 @@
-import React from 'react'
-import Table from './salesTable'
+import React from 'react';
+import SalesTable from './SalesTable';
```

Comment:
```suggestion
import SalesTable from './salesTable';
```

Reply:
file name happens to be camel case

Reply:
Actually, I renamed the file to have consistency with the class name. :)